src/utils/ResizableWithHandle.scss

```diff
@@ -0,0 +1,8 @@
+.resizable-with-handle__handle div  {
+  right: 6px !important;
```

Comment:
❓ how come the `!important` is needed?

Reply:
horrible isn't it :D it's because the package sets the handle styles inline, and this avoided me needing to redefine all the styles, and keep things tidy...I don't think I can set the one style inline because it would get overwritten by the component

Reply:
I'll have a little think, in case there's a tidier approach

Reply:
Ah no worries, if it's needed and the simplest solution then I'm more than happy. Just wondered what the cause was 👍